pilot/cmd/pilot-agent/status/ready/probe.go

```diff
@@ -52,6 +54,10 @@ func (p *Probe) Check() error {
 
 // checkConfigStatus checks to make sure initial configs have been received from Pilot.
 func (p *Probe) checkConfigStatus() error {
+	if p.NoEnvoy {
+		// TODO some way to verify XDS proxy -> control plane works
```

Comment:
we should verify we got a cert?

Reply:
Should the logic be:

```

if grpcEnv {
  checkForBootstrapJSON
  if outputCert {
    checkCert
  }
}
```

I don't want to start changing too much behavior of things not related to proxyless

Reply:
SGTM